### ISSUE TITLE
State is not updated while error is shown on editText

### ISSUE THREAD

**Issue by mlilienberg** [Crysis21/KeyboardHeightProvider#2]
when calling edittext.error = "any text" the error is show as kind of overlay. While such an error is visible, the OnGlobalLayoutListener in KeyboardHeightProvider is not notified. 

This should not be a big issue when using new error approach via TextInputLayout.

**Comment by Crysis21** [comment]
I'm not sure I understood the issue. What was the desired behaviour, that the `KeyboardHeightProvider` should be notified when an error pops up in an `EditText`?

**Comment by mlilienberg** [comment]
The issue is that notifyKeyboardHeightChanged is not called when keyboard is showing or hiding while an ErrorPopup is shown (see Editor.class). Those errors are implemented as PopupWindow and it seems that the overall approach is not working when another PopupWindow is displayed at the same time.

**Comment by Crysis21** [comment]
I'm trying to find a fix for this issue. I tried everything I could've think of, but no success. From what I saw, if I attach the `globalLayoutListeners` again, after the error `PopupWindow` is displayed, I receive the notifications. Also, if you start typing in the `EditText`, the keyboard is computed again. I don't understand why the layout listeners don't report anymore when another window is shown.
